Logging/Entities/Body.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Logging.Entities
+{
+    public class Body
+    {
+        public long bytes { get; set; }
+    }
+}
```

Comment:
Please always add new lines.

I don't understand why it is necessary to define all these classes for logging properties: can't we use a map?

```suggestion
}

```